### DISCUSSION TITLE
For PFF model, what is different by using the grain tracker and by not using  grain tracker

### DISCUSSION THREAD

**Discussion by caoductoan379** [idaholab/moose#16335] [Q&A Modules: Phase field] [answered]
Dear Moose

I am simulating the PFF example. I used a input file with including the grain tracker part and without including the grain tracker part to run program for the same micro structure . I received two different results for crack propagating. 

Could you please let me know  what is different by using the grain tracker and by not using the grain tracker in Moose for PFF model?

Kind regards, 

Toan

**Comment by jiangwen84** [comment, marked as answer]
It will be helpful if you attach your input file. What do you use grain tracker for?

**Reply by caoductoan379** [reply]
Thanks Jiang 

Right now, I understand what is different because i used two constitutive function different 
1. without grain tracker
  [./elasticity_tensor]
    type = ComputeElasticityTensorCP
    C_ijkl = '1.27e5 0.708e5 0.708e5 1.27e5 0.708e5 1.27e5 0.7355e5 0.7355e5 0.7355e5'
    fill_method = symmetric9
    read_prop_user_object = prop_read
  [../]

2. with grain tracker 

  [./ElasticityTensor]
    type = ComputePolycrystalElasticityTensor
    grain_tracker = grain_tracker
    outputs = all
  [../]

Kind regards, 

Toan
